.travis.yml

```diff
@@ -46,9 +46,10 @@ matrix:
     - env:
         - JOB="3.7" ENV_FILE="ci/deps/travis-37.yaml" PATTERN="(not slow and not network and not clipboard)"
 
-    - arch: arm64
-      env:
-        - JOB="3.7, arm64" PYTEST_WORKERS=1 ENV_FILE="ci/deps/travis-37-arm64.yaml" PATTERN="(not slow and not network and not clipboard and not arm_slow)"
+    # Disabled 2020-09-29 due to timeoutshttps://github.com/pandas-dev/pandas/issues/36719
```

Comment:
instead can we move it to allowed failures again

Reply:
wont that still run and cause the Travis turnaround to be super-slow?

Reply:
it will turn the build green as soon as the non allow failure builds succeed